Assets/Editor/TestResponseDeserialization.cs

```diff
@@ -73,7 +73,7 @@ public class TestResponseDeserialization {
 
             Assert.IsNull(response.DataJSON);
             Assert.IsNotNull(response.errors);
-            Assert.AreEqual("Field 'doesntExist' doesn't exist on type 'Shop'", response.errors[0]);
+            Assert.AreEqual("[\"Field 'doesntExist' doesn't exist on type 'Shop'\"]", response.errors);
```

Comment:
In this case it really doesn't matter but the returned list would look something like this:

```
 [
    {
      "message": "Field 'doesntExist' doesn't exist on type 'Shop'",
      "locations": [
        {
          "line": 6,
          "column": 5
        }
      ],
      "fields": [
        "query",
        "shop",
        "doesntExist"
      ]
    }
  ]
```